### PR TITLE
Update documentation in /docs folder.

### PR DESCRIPTION
**Pull request by s-radyuk**
This PR introduces minor changes in documentation contained in `/docs` folder. Mostly typos and invalid links fixing. Also removing excess /docs/README.md file as unnecessary.

### CODE REVIEW

File: docs/jmeter/README.md [hunk 1]
```diff
@@ -42,8 +42,8 @@ List of external JMeter plugins used in Kangal setup:
 * jpgc-fifo [Inter-Thread Communication](https://jmeter-plugins.org/wiki/InterThreadCommunication/)
 * jpgc-functions [Custom JMeter Functions](https://jmeter-plugins.org/wiki/Functions/)
 
-You can also use and modify example test files from [Kangal repository](/examples) as described at [docs/jmeter/writing-tests.md](/docs/jmeter/how-to-write-tests.md).
-Reading the [offical documentation](https://jmeter.apache.org/usermanual/test_plan.html) is strongly recommend to understand major concepts.
+You can also use and modify example test files from [Kangal repository](https://github.com/hellofresh/kangal/tree/master/examples) as described at [docs/jmeter/writing-tests.md](writing-tests.md).
+Reading the [official documentation](https://jmeter.apache.org/usermanual/test_plan.html) is strongly recommend to understand major concepts.
```
Comment (fabioserra) [inline docs/jmeter/README.md:46]: "strongly recommended"

File: docs/jmeter/README.md [hunk 2]
```diff
@@ -65,9 +65,10 @@ JMETER_WORKER_MEMORY_REQUESTS
 
 For the full list of possible environment variables check [Kangal environment variables](../env-vars.md)
 
-More information regarding resource limits and requests can be found in the following page(s):
-- https://kubernetes.io/docs/concepts/configuration/manage-resources-containers/
-- https://cloud.google.com/blog/products/gcp/kubernetes-best-practices-resource-requests-and-limits
+More information regarding resource limits and requests can be found in the following pages:
+
+- [https://kubernetes.io/docs/concepts/configuration/manage-resources-containers/](https://github.com/hellofresh/kangal/tree/master/examples)
```
Comment (fabioserra) [inline docs/jmeter/README.md:70]: I believe the url is wrong no? Also, for the next line.

File: docs/jmeter/writing-tests.md [hunk 3]
```diff
@@ -396,6 +397,8 @@ Some tests require files as images, JAR files, etc. You can provide this from a
 
 The data will be cloned from the bucket to the volume using [Rclone](https://rclone.org/) and will be available to all the pods.
 
-For the full list of possible environment variables check [Kangal environment variables](env-vars.md)
+For the full list of possible environment variables check [Kangal environment variables](../env-vars.md)
```
Comment (fabioserra) [inline docs/jmeter/writing-tests.md:400]: I believe for TechDocs is better to have the fullpath, no?

File: docs/jmeter/writing-tests.md [hunk 4]
```diff
@@ -396,6 +397,8 @@ Some tests require files as images, JAR files, etc. You can provide this from a
...
+
+**Attention**
```
Comment (fabioserra) [inline docs/jmeter/writing-tests.md:402]: Extra space in the end

**Review by fabioserra — CHANGES_REQUESTED** [review]
Nice work! Just a few findings...

File: docs/jmeter/writing-tests.md (hunk 3 above)
Comment (s-radyuk) [reply docs/jmeter/writing-tests.md:400]: It's ok when addressing to the files in the same /docs folder. I testes it locally and worked fine.

File: docs/jmeter/README.md (hunk 2 above)
Comment (s-radyuk) [reply docs/jmeter/README.md]: Good catch! Wrong url in copy buffer 😅

File: docs/jmeter/README.md (hunk 1 above)
Comment (s-radyuk) [reply docs/jmeter/README.md]: 👍

File: docs/jmeter/writing-tests.md (hunk 4 above)
Comment (s-radyuk) [reply docs/jmeter/writing-tests.md]: 👍

**Review by fabioserra — APPROVED** [review]

**Pull request merged**